plugins/polio/api.py

```diff
@@ -394,7 +394,7 @@ def get_round(self: "CampaignViewSet", round: Round, campaign: Campaign, country
         started_at = dt.datetime.strftime(round.started_at, "%Y-%m-%d") if round.started_at is not None else None
         ended_at = dt.datetime.strftime(round.ended_at, "%Y-%m-%d") if round.ended_at is not None else None
         obr_name = campaign.obr_name if campaign.obr_name is not None else ""
-        vacine = self.get_campain_vaccine(round, campaign)
+        vaccines = campaign.vaccines
```

Comment:
this is incorrect when we have separate scopes per round.

You need to check if `separate_scope_per_round` is `True` for the campaign. If it's the case, you need to get the vaccines for the round. If not , you can use the campaign vaccines like you do here

We have this PR https://github.com/BLSQ/iaso/pull/683 that has been merged recently and allows you to use `Round.vaccine_names` to get the vaccines associated with round-level scopes